cypress/integration/styling-and-classes.spec.ts

```diff
@@ -104,4 +104,13 @@ describe('Styling and Classes', () => {
         });
     });
   });
+
+  it.only('should add `.dts-disabled` class to disabled items', () => {
```

Comment:
Nitpick: I have used single quotes in the description of other tests so I'd like to streamline this:
```suggestion
  it.only(`should add 'dts-disabled' to disabled items`, () => {
```